protocol/contracts/libraries/Silo/LibGerminate.sol

```diff
@@ -458,19 +459,19 @@ library LibGerminate {
         if (s.sys.silo.assetSettings[token].milestoneSeason < s.sys.season.current) {
             prevStalkEarnedPerSeason = s.sys.silo.assetSettings[token].stalkEarnedPerSeason;
         } else {
-            int24 deltaStalkEarnedPerSeason = s
+            int64 deltaStalkEarnedPerSeason = s
                 .sys
                 .silo
                 .assetSettings[token]
                 .deltaStalkEarnedPerSeason;
             if (deltaStalkEarnedPerSeason >= 0) {
                 prevStalkEarnedPerSeason =
                     s.sys.silo.assetSettings[token].stalkEarnedPerSeason -
-                    uint32(int32(deltaStalkEarnedPerSeason));
+                    uint32(uint64(deltaStalkEarnedPerSeason));
```

Comment:
why is this casted to a uint64, then a uint32?

Reply:
yes, a leftover from experimenting with different sizes:  https://github.com/BeanstalkFarms/Beanstalk/pull/1092/commits/9249efddefd2f3013c10e468ab03c4b4b1e91ad9#diff-d2261445685829aad9421eae03f9644ae9d11be5713ccc0870497a0be1e58d8aR470